.github/workflows/tox.yml

```diff
@@ -37,18 +37,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ["3.10"]
-        django: [22,32,40,main]
+        python: ["3.8", "3.10", "3.11"]
+        django: ["32", "41", "main"]
```

Comment:
OCD coming into play - if one needs quotes give them all quotes - less confusing for someone new to the project who doesn't know why quotes are / are not required.